dev-util/boost-build/boost-build-1.62.0-r1.ebuild

```diff
@@ -50,10 +50,6 @@ pkg_setup() {
 	fi
 }
 
-src_unpack() {
-	tar xjpf "${DISTDIR}/${A}" boost_${MY_PV}/tools/build || die "unpacking tar failed"
```

Comment:
Are you convinced it's redundant? Because FWICS:
1. default doesn't specify `-p` (not sure if it's really required, though),
2. it unpacks only a subset of the tarball.

Not saying I like the idea of using A like this since it's going to turn into disaster if someone adds another file there. However, I see your change as a regression.